### PR TITLE
Simplified Idna Mapping tables

### PR DESCRIPTION
**Pull request by CAD97**
r? @behnam

### CODE REVIEW

**Comment by CAD97** [comment]
Failure is https://github.com/rust-lang/rust/issues/44208, on master as well.

**Comment by CAD97** [comment]
bors: try

File: gen/src/generate/idna/mapping.rs [hunk 1]
```diff
@@ -65,11 +65,11 @@ impl IdnaMapping {
                 },
             )?;
             if let Some(ref chars) = line.mapping {
-                write!(file, "(&[")?;
+                write!(file, "(\"")?;
                 for char in chars.iter() {
-                    write!(file, "'{}',", char.escape_unicode())?;
+                    write!(file, "{}", char.escape_unicode())?;
```
Comment (behnam) [inline gen/src/generate/idna/mapping.rs]: Maybe add a TODO comment on the for loop to switch to `str::escape_unicode()` when it's stable?

**Comment by behnam** [comment]
bors: r+

**Pull request merged**